components/Layout/Navbar.tsx

```diff
@@ -30,24 +30,26 @@ export const Navbar = (): JSX.Element => {
       </Link>
       <HStack
         spacing={{ base: '16px', lg: '32px' }}
-        display={{ base: 'none', md: 'inherit' }}
+        display={{ base: 'none', lg: 'inherit' }}
```

Comment:
Makes the breakpoint at a higher width so the links don't get squished now that we've added another one